cutlery/_bbpe.pyx

```diff
@@ -51,13 +52,15 @@ cdef class ByteBPEProcessor:
     cdef dict _byte_decoder
     cdef dict _byte_encoder
     cdef object _split_pattern
-    cdef dict _vocab
+    cdef dict _piece_to_id
+    cdef dict _id_to_piece
 
     def __init__(self, vocab: Dict[str, int], merges: List[Tuple[str, str]]):
         self._byte_encoder = bytes_to_unicode()
-        self._byte_decoder = {v: k for k, v in self._byte_encoder.items()}
+        self._byte_decoder = {v.decode("utf-8"): k for k, v in self._byte_encoder.items()}
```

Comment:
Couldn't add the comment to the actual line, but it looks like `bytes_to_unicode()`'s return type annotation should be `Dict[int, bytes]`.

Reply:
Fixed.